CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## vNext
 -
 
+## v0.21.0
```

Comment:
Double check the release checklist (point #9)

Reply:
fixed